### PR TITLE
M365d security-useraccount add display name

### PR DESCRIPTION
**Pull request by amirfeldman**

### CODE REVIEW

File: changelog/Microsoft.M365.Defender.json [hunk 1]
```diff
@@ -1,898 +1,934 @@
...
+        }
+      ],
+      "Id": "2674744f-725e-4e6b-9768-5766b3713d2c",
+      "Cloud": "Prod",
+      "Version": "v1.0",
+      "CreatedDateTime": "2023-01-30T13:00:00.1231234Z",
+      "WorkloadArea": "Security",
+      "SubArea": "Alerts and incidents"
+    },
+    {
```
Comment (Jarbas-MSFT) [inline changelog/Microsoft.M365.Defender.json:897]: Here starts the changelog for this PR. The previous changes are related to a new formatting for this entire JSON object.

File: api-reference/v1.0/resources/security-useraccount.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ---
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/security-useraccount.md:1]: For consistency, please update the response examples in the [LIST](https://learn.microsoft.com/en-us/graph/api/security-list-alerts_v2?view=graph-rest-1.0&tabs=http#response-1) and [GET](https://learn.microsoft.com/en-us/graph/api/security-alert-get?view=graph-rest-1.0&tabs=http#response-1) operations of the [alert](https://learn.microsoft.com/en-us/graph/api/resources/security-alert?view=graph-rest-1.0) resource.


Same for:
- api-reference/beta/resources/security-useraccount.md

File: changelog/Microsoft.M365.Defender.json (hunk 1 above)
Comment (amirfeldman) [reply changelog/Microsoft.M365.Defender.json:897]: is there an AI for me or it's just for future use?

Comment (JarbasHorst) [reply changelog/Microsoft.M365.Defender.json:897]: This is just an information for my colleagues who will do a final review on this PR, so they know what to focus on.

**Comment by Jarbas-MSFT** [comment]
Hi @Lauragra, this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**